sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/statsbeat/StatsbeatModule.java

```diff
@@ -45,7 +45,7 @@ public StatsbeatModule(Consumer<MetadataInstanceResponse> vmMetadataServiceCallb
         featureStatsbeat = new FeatureStatsbeat(customDimensions, FeatureType.FEATURE);
         instrumentationStatsbeat = new FeatureStatsbeat(customDimensions, FeatureType.INSTRUMENTATION);
         azureMetadataService = new AzureMetadataService(attachStatsbeat, customDimensions, vmMetadataServiceCallback);
-        if (RpAttachType.getRpAttachType() == RpAttachType.MANUAL) {
+        if (RpAttachType.getRpAttachType() != RpAttachType.MANUAL) {
             networkStatsbeat = new NetworkStatsbeat(customDimensions);
             nonessentialStatsbeat = new NonessentialStatsbeat(customDimensions);
         } else {
```

Comment:
why don't we send network statsbeat in all cases?

Reply:
we only collect attach and feature for standalone exporter per discussion with ram earlier
do we want to collect all?

Reply:
let's confirm with @ramthi, it's not clear to me why we wouldn't collect network statsbeat for standalone exporter

Reply:
I think this is a misunderstanding that network stats need not be collected from Exporter.  Any and all code we write that helps customers send telemetry should capture (all possible) stats.  Otherwise, we will not get a full picture of what our customers are experiencing.  @heyams, can you pl. point me to a PR where I may have said something that created this confusion, if it was a PR?

Reply:
it wasn't in a pr. I asked you in a meeting. dotnet is the same only collecting attach.

Reply:
i can change it to collect all. i think it's just matter of cost. usage from standalone exporter should be low..

Reply:
this is azure-monitor-opentelemetry-exporter not the breeze exporter. just want to remind you. "exporter' alone is confusing.

Reply:
> it wasn't in a pr. I asked you in a meeting. dotnet is the same only collecting attach.

Thanks for clarifying. I think this is not correct.  I will talk to .NET folks.

Reply:
> this is azure-monitor-opentelemetry-exporter not the breeze exporter. just want to remind you. "exporter' alone is confusing.

azure-monitor-opentelemetry-exporter sends to Breeze right?  Now, I'm confused how this is different than Breeze exporter :)  We can perhaps chat on Teams about this.  Let me know.

Reply:
one more thing, we don't want to collect non-essential statsbeat, correct?

Reply:
let's chat offline, in the future we should put this kind of decision in a telemetry-collection-spec so that we don't need to go back and forth @ramthi

Reply:
discussed with @heyams, going to merge as-is to b/c blocking Java agent release, and we will follow-up after

Reply:
Yup, added a todo for now